dashboard/lib/pd/payment/payment_calculator.rb

```diff
@@ -42,8 +42,8 @@ def calculate_workshop_payment(workshop)
       payment_sum = payment_term.fixed_payment || 0
 
       if payment_term.minimum_attendees_for_payment? &&
-          attending_teachers.size < payment_term.minimum_attendees_for_payment
-        return payment_sum
+          workshop.enrollments.size < payment_term.minimum_attendees_for_payment
```

Comment:
if this is about enrollments now, should we rename the field `mimimum_enrollments_for_payment`?